clerk/types/codec.go

```diff
@@ -6,12 +6,15 @@ import (
 
 // RegisterCodec registers concrete types on codec codec
 func RegisterCodec(cdc *codec.Codec) {
-	cdc.RegisterConcrete(MsgEventRecord{}, "cosmos-sdk/MsgEventRecord", nil)
+	cdc.RegisterConcrete(MsgEventRecord{}, "clerk/MsgEventRecord", nil)
```

Comment:
Did you test it?

Reply:
@mankenavenkatesh Let's keep `cosmos-sdk/` for now since it will break current v1.